source/_integrations/homeassistant.markdown

```diff
@@ -3,6 +3,116 @@ title: "Core integration"
+
+#### Service `homeassistant.stop`
+
+Stops the Home Assistant instance. Home Assistant must be restarted from the Host device to run again.
+
+#### Service `homeassistant.set_location`
+
+Update the location of the Home Assistant default zone (usually "Home").
+
+| Service data attribute    | Optional | Description                                           |
+|---------------------------|----------|-------------------------------------------------------|
+| `latitude`                |       no | Latitude of your location.                            |
+| `longitude`               |       no | Longitude of your location.                           |
+
+##### Example
+
+```yaml
+action:
+  service: homeassistant.set_location
+  data:
+    latitude: 32.87336
+    longitude: 117.22743
+```
+
+#### Service `homeassistant.toggle` 
+
+Generic service to toggle devices on/off under any domain. Same usage as the light.turn_on, switch.turn_on, etc. services.
+
+| Service data attribute    | Optional | Description                                           |
+|---------------------------|----------|-------------------------------------------------------|
+| `entity_id`               |       no | The entity_id of the device to toggle on/off.         |
+
+##### Example
+
+```yaml
+action:
+  service: homeassistant.toggle
+  data:
+    entity_id: light.living_room
+```
+
+#### Service `homeassistant.turn_on` 
+
+Generic service to turn devices on under any domain. Same usage as the light.turn_on, switch.turn_on, etc. services.
+
+| Service data attribute    | Optional | Description                                           |
+|---------------------------|----------|-------------------------------------------------------|
+| `entity_id`               |       no | The entity_id of the device to turn on.               |
+
+##### Example
+
+```yaml
+action:
+  service: homeassistant.turn_on
+  data:
+    entity_id: light.living_room
+```
+
+#### Service `homeassistant.turn_off` 
+
+Generic service to turn devices off under any domain. Same usage as the light.turn_on, switch.turn_on, etc. services.
+
+| Service data attribute    | Optional | Description                                           |
+|---------------------------|----------|-------------------------------------------------------|
+| `entity_id`               |       no | The entity_id of the device to turn off.              |
+
+##### Example
+
+```yaml
+action:
+  service: homeassistant.turn_off
+  data:
+    entity_id: light.living_room
+```
+
+#### Service `homeassistant.update_entity` 
+
+Force one or more entities to update its data rather than wait for the next scheduled update.
+
+| Service data attribute    | Optional | Description                                           |
+|---------------------------|----------|-------------------------------------------------------|
+| `entity_id`               |       no | One or multiple entity_ids to update. Can be a list.  |
```

Comment:
```suggestion
| `entity_id`               |       no | One or multiple entity_ids to update. It can be a list.  |
```

Reply:
pulled from [services.yaml](https://github.com/home-assistant/home-assistant/blob/master/homeassistant/components/homeassistant/services.yaml#L48).
@frenck Does the origin text need to be updated as well?

Reply:
🤷‍♂ I've reviewed the current PR, not the entire site.